omod/src/main/webapp/pages/markPatientDead.gsp

```diff
@@ -60,7 +79,7 @@ ${ui.includeFragment("coreapps", "patientHeader", [patient: patient])}
                 <span>${ui.message("aijar.markpatientdeceased.causeofdeath")} (${ui.message("emr.formValidation.messages.requiredField.label")})</span>
             </label>
             <select name="causeOfDeath" id="cause-of-death">
-                <option>Select Cause Of Death</option>
+                <option value="null">Select Cause Of Death</option>
```

Comment:
@slubwama Check what is used around the rest of OpenMRS code, I suspect that it is an empty string instead of the word null